maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/AssessmentSummaryService.java

```diff
@@ -38,4 +42,21 @@ public void updateApplication(ApplicationDTO application, AssessmentSummaryDTO s
                         }, () -> assessmentSummary.add(summaryDTO)
                 );
     }
-}
+
+    public AssessmentSummaryDTO getSummary(FinancialAssessmentDTO financialAssessmentDTO) {
+        boolean isFullAssessment = AssessmentType.FULL.equals(AssessmentType.getFrom(financialAssessmentDTO.getAssessmentType()));
+        String status = isFullAssessment ? financialAssessmentDTO.getFassFullStatus() : financialAssessmentDTO.getFassInitStatus();
+        String type = isFullAssessment ? "Full Means Test" : "Initial Assessment";
+        String result = isFullAssessment ? financialAssessmentDTO.getFullResult() : financialAssessmentDTO.getInitResult();
+        Date assessmentDate = isFullAssessment ? DateUtil.toDate(financialAssessmentDTO.getFullAssessmentDate()) : DateUtil.toDate(financialAssessmentDTO.getInitialAssessmentDate());
```

Comment:
I think for the sake of readability, an if/else would be better here, rather than several ternary statements.

Reply:
Originally had it as if/else, the AI simplify recommended ternary statements :)  
I guess this is okay?

Reply:
I really think it would help to improve the readability here, so I'd like to see thid changed. Maybe the AI is going for something a little bit different? Copilot seems to like shortening things as much as possible.